pkgs/by-name/br/brickstore/package.nix

```
@@ -0,0 +1,101 @@
+{
+  lib,
+  stdenv,
+  qt6,
+  libsForQt5,
+  fetchFromGitHub,
+  gst_all_1,
+  cmake,
+  libglvnd,
+  tbb,
+  ninja,
+  pkg-config,
+}:
+let
+  inherit (libsForQt5) qcoro;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "brickstore";
+  version = "0-unstable-2024-05-02";
+
+  src = fetchFromGitHub {
+    owner = "rgriebl";
+    repo = finalAttrs.pname;
+    rev = "v2024.5.2";
+    hash = "sha256-Bu9oNbZm3lx/CfYAReHyWe/kW+kaefDWeBtWLHOCORU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    libglvnd
+    ninja
+    pkg-config
+    qcoro
+    qt6.qtdoc
+    qt6.qtdeclarative
+    qt6.qtimageformats
+    qt6.qtmultimedia
+    qt6.qtquick3d
+    qt6.qtquicktimeline
+    qt6.qtshadertools
+    qt6.qttools
+    qt6.qtwayland
+    qt6.wrapQtAppsHook
+    tbb
+  ];
+
+  preConfigure = ''
+    # deletes line https://github.com/rgriebl/brickstore/blob/1e62a8bdf7da848a0033b8647d3711d95a3afd8d/cmake/BuildQCoro.cmake#L21
+    # This isn't good but not sure how to do it via substituteInPlace
+    echo "#########################"
+    echo "#########################"
+    echo "#########################"
+    echo "#########################"
+    echo "#########################"
+    echo "#########################"
+    echo "#########################"
+    echo "#########################"
+    sed -i '/^)$/d' cmake/BuildQCoro.cmake
```

```suggestion
  preConfigure = ''
    # deletes line https://github.com/rgriebl/brickstore/blob/1e62a8bdf7da848a0033b8647d3711d95a3afd8d/cmake/BuildQCoro.cmake#L21
    # This isn't good but not sure how to do it via substituteInPlace
    sed -i '/^)$/d' cmake/BuildQCoro.cmake
```

Make a new commit with the changes and squash it using `git rebase -i HEAD~2`.